arch/risc-v/src/litex/litex_cache.c

```diff
@@ -0,0 +1,88 @@
+/****************************************************************************
+ * arch/risc-v/src/litex/litex_cache.c
```

Comment:
Maybe we can create `arch/risc-v/src/litex/litex_cache.S` instead? I mean that there are only inline ASM calls here

Reply:
Ok. Sounds good.
Converted to .S